django/santropolFeast/member/models.py

```diff
@@ -209,3 +209,21 @@ class Meta:
         related_name='allergic_clients',
         blank=True
     )
+
+
+class Referencing (models.Model):
+
+    class Meta:
+        verbose_name_plural = _('referents')
+
+    referent = models.ForeignKey('member.Member',
+                                 verbose_name=_('member'))
+
+    client = models.ForeignKey('member.Client',
+                               verbose_name=_('client'))
+
+    referral_reason = models.TextField(
+            verbose_name=_("referral_reason"))
+
+    date = models.DateField(verbose_name=_("referral_date"),
```

Comment:
Why you set ```auto_now=False``` ? It can be ok, but i think it's not a bad things if we set a default value.